R/musMusculus/GSE113069/GSE113069.r

```diff
@@ -0,0 +1,29 @@
+library(dplyr)
+library(Seurat)
+library(patchwork)
+library(clustifyr)
+library(tidyverse)
+library(digest)
+library(here)
+
+mat_subregions <- read_tsv("ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE113nnn/GSE113069/suppl/GSE113069_combinedTables.txt.gz")
+mat_subregions <- mat_subregions %>%
+  # as.data.frame() %>%
+  column_to_rownames('B11__d1')
+#  as.matrix() %>%
+#  t()
+mat_subregions[1:5, 1:5]
+
+meta_subregions <- read_tsv("ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE113nnn/GSE113069/suppl/GSE113069_metadata.txt.gz")
+meta_subregions
+ncol(mat_subregions)
+
+source("~/Reference-Matrix-Generation/R/utils/utils.r")
+checkRawCounts(as.matrix(mat_subregions))
+
+GSE113069Normalized <- NormalizeData(mat_subregions)
+
+new_ref_matrix <- average_clusters(mat = GSE113069Normalized, metadata = meta_subregions$note2, if_log = FALSE)
```

Comment:
This give a warning: "Warning message:
In split.default(colnames(mat), cluster_info) :
  data length is not a multiple of split variable", which is due to not having the same number of cells in the mat vs meta. Maybe this should trigger an error instead in clustifyr.

add this:
```
meta_subregions <- meta_subregions %>% mutate(id = str_remove(str_c(id, file, sep = "__"), "\\.dat")) %>% column_to_rownames("id")
meta_subregions2 <- meta_subregions[colnames(mat_subregions),]
```

Reply:
@raysinensis See commit 364f2c5